app/.env.example

```
@@ -1,5 +1,6 @@
+Environment=test
 SUPABASE_URL=https://supabase_url.supabase.co
-SUPABASE_KEY=SUPABASE_SERVIVCE_KEY
+SUPABASE_KEY=SUPABASE_SERVICE_KEY
 
 embedding_model=EMBEDDING_MODEL
```

dotenv-linter (3.3.0)

[warning] [LowercaseKey] The embedding_model key should be in uppercase

(LowercaseKey)